vasern/src/core/vasern-parser/Parser.js

```diff
@@ -32,7 +30,7 @@ const Parser = {
 
     switch (dataType) {
       case "string":
-        return val && val.replace ? val.replace(/\u00A0n/g, "\n") : "";
+        return val && val.replace ? val.replace(/\u00A0n/g, "\n") : undefined;
```

Comment:
Why does it now give undefined instead of an empty string?

Reply:
Ah, good point.

I'm thinking, if the value is `undefined`, store it as `undefined`. Then when getting the record, if the string is `""`, return as it is. If it is `undefined` (as a string) return as `undefined`.

Though, there is one case (maybe rare) that the string is actually `undefined`.

What do you think?

Reply:
I get your point. I would let it as undefined. I think it is more clear.
On the other hand, all other properties get empty strings, if they are undefined, right? So maybe it gets a little bit confusing (I would prefer always undefined if no value is given).